commands/addResourceExtended.ts

```diff
@@ -1,30 +1,52 @@
 import { SlashCommandBuilder } from '@discordjs/builders';
```

Comment:
FWIW my vote is with `addResource` vs this DM style approach.

Reply:
I will remove the code related to the DM style. I also prefer the interaction based approach more. I didn't know how to accomplish it in that manner until @crondinini show us all how with the /add-author flow.

Reply:
Shoutout to @crondinini :D